helpers/validators.helpers.ts

```diff
@@ -58,6 +58,12 @@ export const bytes = <ValidationRule>(
 )
 export const integer = <ValidationRule>withI18nMessage(_integer)
 
+export const contractFuncName = <ValidationRule>(
+  withI18nMessage(value =>
```

Comment:
can it validate ref? vuelidate validators can

Reply:
yes, discussed